app/controllers/application_controller/buttons.rb

```diff
@@ -838,6 +852,13 @@ def button_valid?(button_hash = @edit[:new])
 
     validate_playbook_button(button_hash) if button_hash[:button_type] == "ansible_playbook"
 
+    if button_hash[:object_request].blank?
+      add_flash(_("Request is required"), :error)
+      if @flash_array.length == 1
```

Comment:
@ZitaNemeckova what happens when flash message is also added on https://github.com/ManageIQ/manageiq-ui-classic/pull/5873/files#diff-65512d4378cfa992fb57fbceaaf14c59L823

Reply:
Fixed so it switches for `Starting Process is required` error as well.

Reply:
Missed that error message. Fixed so it switches the tab as well.